<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Test Ti 4500 DLP Controls</string>
  </property>
  <widget class="QPushButton" name="fullScreenBut">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Toggle Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBut">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FPS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fpsLbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame time (ms):</string>
   </property>
  </widget>
  <widget class="QLabel" name="renderTimeLbl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>361</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Frame Speedup Strategy</string>
   </property>
   <widget class="QRadioButton" name="noneRadio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioButs</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="octupleRadio">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>8x FPS, 3-bit gray</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioButs</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="tripleRadio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3x FPS (StimGL)</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioButs</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="dodekaQuadrupleRadio">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>24x FPS, 1-bit monocrome</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">radioButs</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pauseBut">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>Toggle Pause</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="radioButs"/>
 </buttongroups>
</ui>
